tdvt/tdvt/tabquery.py

```diff
@@ -41,11 +41,13 @@ def build_tabquery_command_line(work):
     cmdline = tb.build_tabquery_command_line(work)
     return cmdline
 
-def build_connectors_test_tabquery_command_line(connTestName, connTestFileName):
+def build_connectors_test_tabquery_command_line(connTestName, connTestFileName, connTestPasswordFile):
```

Comment:
```suggestion
def build_connectors_test_tabquery_command_line(connTestName, connTestFileName, connTestPasswordFile=None):
```
I think we need this to default to None otherwise it'll blow up without a password file.

Reply:
I tested with and without the password file. The current code works correctly